File: Bankdata/Bankdata/Models/Transaction.cs
namespace Bankdata.Models
{
    public class Transaction
    {
        public int TransactionId { get; set; }
        public DateTime Modified { get; set; }
        public int ReceiverID { get; set; }
        public int AccountID { get; set; }
        public double Amount { get; set; } = 0;
        public int TransactionID { get; set; }
    }
}

File: BankDataFront/src/stores/Transaction.ts
import { defineStore } from "pinia";
import { instance } from '../network/network.ts'

export interface Transaction {
    ProviderId: number;
    ReceiverId: number;
    TransactionId: number;
    TransferSum: number;
    DateTime: number;
}

export const useTransactionStore = defineStore({
    id: "Transaction",
    state: (): Transaction => ({
        ProviderId: 0,
        ReceiverId: 0,
        TransactionId: 0,
        TransferSum: 0,
        DateTime: Date.now(),
    }),
    actions: {
        
    }
})